dpctl/tensor/CMakeLists.txt

```diff
@@ -188,12 +188,20 @@ foreach(_src_fn ${_no_fast_math_sources})
   )
 endforeach()
 if (UNIX)
-  set_source_files_properties(
-    ${CMAKE_CURRENT_SOURCE_DIR}/libtensor/source/elementwise_functions/abs.cpp
-    ${CMAKE_CURRENT_SOURCE_DIR}/libtensor/source/elementwise_functions/sqrt.cpp
-    PROPERTIES COMPILE_DEFINITIONS "USE_STD_ABS_FOR_COMPLEX_TYPES;USE_STD_SQRT_FOR_COMPLEX_TYPES")
+  set(_compiler_definitions "USE_SYCL_FOR_COMPLEX_TYPES")
+else()
+  set(_compiler_definitions "USE_SYCL_FOR_COMPLEX_TYPES")
```

Comment:
This conditional seems redundant. Maybe it's not supposed to use "USE_SYCL_FOR_COMPLEX_TYPES" if not Unix?

Reply:
It is supposed to be used on both platforms, but the `if` exists because different set of preprocessor variables used to be set on Windows and Linux. I can simplify the cmake script.